src/ci/docker/host-x86_64/mingw-check/Dockerfile

```diff
@@ -41,6 +41,10 @@ COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/
 ENV RUN_CHECK_WITH_PARALLEL_QUERIES 1
 
 ENV SCRIPT python3 ../x.py --stage 2 test src/tools/expand-yaml-anchors && \
+           # Check library crates on all tier 1 targets.
+           # We disable optimized compiler built-ins because that requires a C toolchain for the target.
+           # We also skip the x86_64-unknown-linux-gnu target as it is well-tested by other jobs.
+           python3 ../x.py check --stage 0 --set build.optimized-compiler-builtins=false core alloc std --target=aarch64-unknown-linux-gnu,i686-pc-windows-msvc,i686-unknown-linux-gnu,x86_64-apple-darwin,x86_64-pc-windows-gnu,x86_64-pc-windows-msvc && \
```

Comment:
I guess this isn't including libtest and proc_macro, but those seem less likely to be target-dependent...

Reply:
Yeah. I was going by what tends to fail after an r+. Which is admittedly anecdotal and prone to biasing towards where code changes more frequently. But I figured that if it comes up then it's easy enough to add additional crates.